src/resources/authoring/gv2.js

```diff
@@ -86,14 +86,16 @@ window.GV2Authoring = [
       },
       "targetDrakes": [{},{},{}],
       "hiddenAlleles": "t,tk,h,c,a,b,d,bog,rh"
-    },
+    }
+  ],
+  [
     {
       "template": "GenomePlayground",
       "initialDrake": {
         "alleles": "a:T,b:T,a:w,b:w,a:h,b:h,a:A1,b:A1,a:C,b:C,a:B,b:B,a:D,b:D,a:rh,b:rh,a:Bog,b:Bog",
         "sex": 1
       },
       "hiddenAlleles": "t,tk,h,c,a,b,d,bog,rh"
-    }
+    },
```

Comment:
1. I don't think any browsers we care about would notice. (As I recall, IE8 was the last browser to have that problem.)
2. Even so, I generally try to avoid it as a matter of general code hygiene, so good catch.
3. ESLint must have a setting to catch extraneous commas.
4. It's already been changed in subsequent PRs to go on to Egg Game III rather than circling back to the Genome Playground.